nodejs/eks/cluster.ts

```diff
@@ -484,13 +512,82 @@ export interface ClusterOptions {
     vpcId?: pulumi.Input<string>;
 
     /**
-     * The subnets to attach to the EKS cluster. If either vpcId or subnetIds is unset, the cluster will use the
-     * default VPC's subnets. If the list of subnets includes both public and private subnets, the Kubernetes API
-     * server and the worker nodes will only be attached to the private subnets. See
-     * https://docs.aws.amazon.com/eks/latest/userguide/network_reqs.html for more details.
+     * The set of subnets to use on the EKS cluster.
+     * These subnets are automatically tagged by EKS for Kubernetes purposes.
+     *
+     * Note, `vpcId` must be set. If not, then the cluster will use the AWS
+     * account's default VPC subnets.
+     *
+     * If the list of subnets includes both public and private subnets, the worker
+     * nodes will only be attached to the private subnets, and the public
+     * subnets will be used for internet-facing load balancers.
+     *
+     * See for more details: https://docs.aws.amazon.com/eks/latest/userguide/network_reqs.html.
+     *
+     * Note, the use of `subnetIds`, and the combination of `publicSubnetIds`
+     * and/or `privateSubnetIds` is mutually exclusive. The former is the
```

Comment:
Might reword this to something like "This option will be deprecated in a future release." The former/latter wording is a little tricky to parse.

Reply:
I'm not sure we need to say it will be deprecated, or that there is "old"/"new" functionality.  We can just be fact based about what the legal combinations are and what they do.  If we want to say "passing publicSubnetIds and privateSubnetsIds is encouraged" that seems fine.